tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -470,6 +484,7 @@
           - openstack
           servicesOverride:
           - pre-adoption-validation
+          - reboot-strategy
```

Comment:
I don't think we should be diverging from the docs like this. Let's just put the os-reboot role back into the service list on the NodeSet, both in tests and in docs?

Reply:
ack, I will put it back into pre-validation